cmds/installer.go

```diff
@@ -346,7 +378,7 @@ func newService(namespace, addr string) runtime.Object {
 			ClusterIP: host,
 			Ports: []core.ServicePort{
 				{
-					Name:       "api",
+					Name:       "service-api",
```

Comment:
Why rename?

Reply:
i renamed it because container port name and service port name is same, but port is different. So, when i testing it in minikube, guard server didn't get any request. In kubernetes logs i found the 'connection refused error'.

Reply:
should be fixed now